elide-core/src/test/java/com/yahoo/elide/core/LifeCycleTest.java

```diff
@@ -280,34 +280,34 @@ public void testCreate() {
         resource.setValueChecked("title", "should not affect calls since this is create!");
         resource.setValueChecked("genre", "boring books");
         assertNotNull(resource);
-        verify(book, never()).onCreateBook(scope);
+        verify(book, never()).onCreatePreSecurity(scope);
         verify(book, never()).checkPermission(scope);
 
         scope.runQueuedPreSecurityTriggers();
-        verify(book, times(1)).onCreateBook(scope);
-        verify(book, never()).onDeleteBook(scope);
-        verify(book, never()).onUpdateTitle(scope);
-        verify(book, never()).onCreateBookPreCommit(eq(scope), any());
-        verify(book, times(1)).preRead(scope);
+        verify(book, times(1)).onCreatePreSecurity(scope);
+        verify(book, never()).onDeletePreSecurity(scope);
+        verify(book, never()).onUpdatePreSecurityTitle(scope);
+        verify(book, never()).onCreatePreCommit(eq(scope), any());
+        verify(book, times(1)).onReadPreSecurity(scope);
         verify(book, never()).checkPermission(scope);
 
         scope.runQueuedPreCommitTriggers();
-        verify(book, times(1)).preCreateBook(scope);
-        verify(book, never()).preDeleteBook(scope);
-        verify(book, never()).preUpdateTitle(scope);
-        verify(book, times(2)).onCreateBookPreCommit(eq(scope), any());
-        verify(book, times(1)).preCommitRead(scope);
+        verify(book, times(1)).onCreatePreCommit(scope);
+        verify(book, never()).onDeletePreCommit(scope);
+        verify(book, never()).onUpdatePreCommitTitle(scope);
+        verify(book, times(2)).onCreatePreCommit(eq(scope), any());
```

Comment:
Can we rename this onCreatePreCommitStar or something?

Reply:
Done